frontend/javascripts/viewer/view/ai_jobs/ai_jobs_drawer.tsx

```diff
@@ -0,0 +1,65 @@
+import { Drawer, Tabs } from "antd";
+import { useWkSelector } from "libs/react_hooks";
+import { useCallback } from "react";
+import { useDispatch } from "react-redux";
+import { setAIJobDrawerStateAction } from "viewer/model/actions/ui_actions";
+import { AiImageAlignmentJob } from "./alignment/ai_image_alignment_job";
+import type { StartAiJobDrawerState } from "./constants";
+import { AiImageSegmentationJob } from "./run_ai_model/ai_image_segmentation_job";
+import { AiModelTrainingJob } from "./train_ai_model/ai_training_job";
+
+export const AiJobsDrawer = ({ isOpen }: { isOpen: boolean }) => {
+  const dispatch = useDispatch();
+  const isSuperUser = useWkSelector((state) => state.activeUser?.isSuperUser);
+  const ai_job_drawer_state = useWkSelector((state) => state.uiInformation.aIJobDrawerState);
+
+  const handleChange = useCallback(
+    (key: string) => {
+      dispatch(setAIJobDrawerStateAction(key as StartAiJobDrawerState));
+    },
+    [dispatch],
+  );
+
+  const handleClose = useCallback(() => {
+    dispatch(setAIJobDrawerStateAction("invisible"));
+  }, [dispatch]);
+
+  const items = [
+    {
+      label: "Image Segmentation",
+      key: "open_ai_inference",
+      children: <AiImageSegmentationJob />,
+    },
+    ...(isSuperUser
+      ? [
+          {
+            label: "Model Training",
+            key: "open_ai_training",
+            children: <AiModelTrainingJob />,
+          },
+        ]
+      : []),
+    {
+      label: "Image Alignment",
+      key: "open_ai_alignment",
+      children: <AiImageAlignmentJob />,
+    },
+  ];
+
+  const activeKey =
+    !isSuperUser && ai_job_drawer_state === "open_ai_training"
+      ? "open_ai_inference"
+      : ai_job_drawer_state;
```

Comment:
isn't this equivalent to `const activeKey = ai_job_drawer_state;`?

Reply:
No, for the time being non-superusers are not allowed to open the training job drawer.